src/olympia/addons/serializers.py

```diff
@@ -1648,3 +1649,14 @@ def get_replacement(self, obj):
                 coll_match.group('user_id'), coll_match.group('coll_slug')
             )
         return []
+
+
+class AddonBrowserMappingSerializer(AMOModelSerializer):
+    addon_guid = serializers.CharField(source='addon__guid')
```

Comment:
Add a comment explaining that this is coming from an annotation (since it can be a little surprising)